### DISCUSSION TITLE
Home Assistant/MariaDB for for Frigate database a good idea?

### DISCUSSION THREAD

**Discussion by heyitsyang** [blakeblackshear/frigate#4176] [General]
I have installed the MariaDB add-on in Home Assistant as a better performing replacement for SQLite.  I'm just now installing Frigate.

Has anyone else used the MariaDB HA add-on as the Frigate DB?
Are there drawbacks to doing so?

**Comment by NickM-27** [comment]
Frigate doesn't support using any DB other than the built in sqlite

**Reply by heyitsyang** [reply]
@NickM-27,  thanks for your reply.  Help me understand.  Does Frigate install it's own instance of SQLite or do I need to re-install the SQLite add-on in Home Assistant?

**Reply by NickM-27** [reply]
The frigate database is fully self contained, you should not interact with it in any way directly
